rateslib/periods.py

```diff
@@ -1098,9 +1098,10 @@ def _ibor_rate_from_line_curve(self, curve: LineCurve):
         return curve[fixing_date] + self.float_spread / 100
 
     def _rfr_rate_from_df_curve(self, curve: Curve):
+        if self.start < curve.node_dates[0]:
```

Comment:
This is the key change in this PR. Resetting the rate calculation when the period start does not align with the curve